lib/s3routes/routesUtils.js

```diff
@@ -385,11 +385,15 @@ function _responseBody(responseBackend, errCode, payload, response, log,
     return undefined;
 }
 
-function _contentLengthMatchesLocations(contentLength, dataLocations) {
-    const sumSizes = dataLocations.reduce(
-        (sum, location) => (sum !== undefined && location.size ?
+function _computeContentLengthFromLocation(dataLocations) {
+    return dataLocations.reduce(
+        (sum, location) => (sum !== undefined && typeof location.size === 'number' ?
```

Comment:
wouldn't we not need to parseInt if it was already a number?

Reply:
it also works with number :)

Reply:
It should not be needed if it is a number, I would change it to just `sum + location.size` but it might be too much to change.

Reply:
that's right, thinking about it, i will check for number or string, to avoid any side effect